distribution/lib/Standard/Base/0.0.0-dev/src/Errors/Common.enso

```diff
@@ -494,3 +494,19 @@ type Loss_Of_Numeric_Precision
     to_text : Text
     to_text self =
         "(Loss_Of_Numeric_Precision.Warning (original_value = "+self.original_value.to_text+") (new_value = " + self.new_value.to_text+"))"
+
+## Indicates that a computed or provided value is outside the allowable range.
+type Out_Of_Range
+    ## PRIVATE
+    Error value
+
+    ## PRIVATE
+       Create a human-readable version of the error.
+    to_display_text : Text
+    to_display_text self =
+        "Value out of range: "+self.value.to_text
```

Comment:
IMHO it would be good to indicate what range we exceed, e.g.
`Value out of representable Float range: ...` in this case.
Do you think we can add this to the error info?

Reply:
Done in #9586.